docs/content/docs/maintenance-actions/_index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Maintenance Actions
```

Comment:
Maybe just `Tuning`?
I think `Actions` can not cover all in here.

Reply:
Just `Maintenance`.